app/views/events/new.html.erb

```diff
@@ -1,8 +1,7 @@
 <script src="event.js"></script>
 
-<%- model_class = Event -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => :'helpers.titles.new', :model => model_class.model_name.human.titleize %></h1>
+  <h1><%= t '.title', default: :'helpers.titles.new', model: @event.model_name.human.titleize %></h1>
```

Comment:
is `@event` really a fitting name here? (When i see it correctly, there is a class inside this variable --> maybe call it `@event_type` or `@event_class` instead)

Reply:
The `@event` is the event that is being created in the `new` action of the respective controller. `model_name` is defined on the class and any instance.